development/nautobot_config.py

```diff
@@ -249,6 +249,7 @@ def is_truthy(arg):
         "barchart_bar_width": float(os.environ.get("BARCHART_BAR_WIDTH", 0.1)),
         "barchart_width": int(os.environ.get("BARCHART_WIDTH", 12)),
         "barchart_height": int(os.environ.get("BARCHART_HEIGHT", 5)),
+        "device_platform_conversion": {"cisco":{"ios":["ios", "ios-xr", "ios-xe"]}}
```

Comment:
With change in platform recognition this line no longer needed.  Will remove with commit to patch for 1.3.3

Reply:
Removed conversion;

Resolved the 1.3.3 issue with job
1.3.3 did not like:
```
published_date=dateutil.parser.parse(info['published_date']),
last_modified_date=dateutil.parser.parse(info['modified_date']),
```

Modified to:
```
published_date=date.fromisoformat(info['published_date'][0:10]),
last_modified_date=date.fromisoformat(info['modified_date'][0:10])
```